hazelcast/src/main/java/com/hazelcast/cache/impl/journal/RingbufferCacheEventJournalImpl.java

```diff
@@ -93,7 +96,14 @@ public long oldestSequence(ObjectNamespace namespace, int partitionId) {
 
     @Override
     public void destroy(ObjectNamespace namespace, int partitionId) {
-        getRingbufferService().destroyContainer(partitionId, namespace);
+        final RingbufferService service;
+        try {
+            service = getRingbufferService();
+        } catch (Exception e) {
+            logger.fine("Could not destroy event journal " + namespace, e);
+            return;
+        }
+        service.destroyContainer(partitionId, namespace);
```

Comment:
The error message `"Could not destroy event journal " + namespace` doesn't really match the code `getRingbufferService()`. It's more like `"Could not retrieve Ringbuffer to destroy event journal " + namespace`.

So shouldn't we move this into the try-catch block? This will also get rid of the `return` in the catch block.